types/apidoc/apidoc-tests.ts

```diff
@@ -0,0 +1,22 @@
+import apidoc from 'apidoc';
+
+const apidocOutput = apidoc({
```

Comment:
> I don't know how to fix the error. The package does not have a default export.

The library really does not have default export:

```js
var apidoc = require("apidoc")

console.log(apidoc); // Object {createDoc: function()}
```

```suggestion
import {createDoc} from 'apidoc';

const apidocOutput = createDoc({
```

Reply:
Looks like it worked. Thanks!